source/host/KTestSystem.cc

```diff
@@ -254,10 +281,13 @@ void KTestSystem::startNextTestCase() {
 
     ASSERT(m_stream.read_u32() == m_stream.index());
 
-    // If we're in Ghost mode instead of Suite mode, then target the total framecount of the KRKG.
+    // If we're in Ghost mode instead of Suite mode and framecount not specified, then target the
+    // total framecount of the KRKG.
     if (m_testMode == Host::EOption::Ghost) {
         ASSERT(m_testCases.size() == 1);
-        m_testCases.front().targetFrame = m_frameCount;
+        if (m_testCases.front().targetFrame == 0) {
+            m_testCases.front().targetFrame = m_frameCount;
+        }
```

Comment:
What happens if I pass `-f 2000` on a ghost that's only 1000 frames? I'd like to know what the behavior is in this edge case. I wonder if we should instead have something like:
```c++
auto &front = m_testCases.front();
front.targetFrame = std::min(front.targetFrame, m_frameCount);
```

Reply:
Kinoko handles this already, it errors if the targetFrame is larger than the m_framecount

Reply:
For now I just added the std::min method